### PR TITLE
feat(ColorsBlock): design review changes

### PR DESCRIPTION
**Pull request by leko91**
As requested in: https://app.clickup.com/t/2y9jphr.

Note: currently missing ~~two~~ ~~one~~ zero fixes from the ticket 🥳

### CODE REVIEW

File: packages/colors-block/src/components/drops/DropsItem.tsx [hunk 1]
```diff
@@ -63,13 +64,18 @@ export const DropsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
                 <Tooltip
                     withArrow
                     position={TooltipPosition.Right}
-                    hoverDelay={0}
-                    content={<TooltipContent colorValue={mappedFirstColorSpace.value ?? ''} status={status} />}
+                    content={
+                        <TooltipContent
+                            colorName={color.name || ''}
```
Comment (celine-s) [inline packages/colors-block/src/components/drops/DropsItem.tsx:69]:
```suggestion
                            colorName={color.name ?? ''}
```

File: packages/colors-block/src/components/drops/DropsItem.tsx [hunk 2]
```diff
@@ -87,50 +93,69 @@ export const DropsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
...
-                                {mappedColorSpace.label}
-                            </div>
+                    return (
+                        <div
+                            data-test-id="color-space"
+                            key={colorSpaceId}
+                            className={merge([
+                                'tw-flex tw-justify-center tw-items-center tw-mb-1 last:tw-mb-0',
+                                isEditing && 'tw-w-full',
+                                !isEditing && !mappedColorSpace.value && 'tw-mb-0',
```
Comment (celine-s) [inline packages/colors-block/src/components/drops/DropsItem.tsx:107]: not sure if condition is still the same but would return alw a string
```suggestion
                                (isEditing || !mappedColorSpace.value) ? '' : 'tw-mb-0',
```

File: packages/colors-block/src/components/drops/DropsItem.tsx [hunk 3]
```diff
@@ -87,50 +93,69 @@ export const DropsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
...
+                            <div
+                                className={merge([
+                                    'tw-flex tw-items-center',
+                                    isEditing && 'tw-justify-between tw-w-full',
+                                ])}
+                            >
+                                <div
+                                    className={merge([
+                                        'tw-flex-[0_0_auto] tw-mr-1.5 tw-text-xs tw-text-black-70',
+                                        !isEditing && !mappedColorSpace.value && 'tw-hidden',
```
Comment (celine-s) [inline packages/colors-block/src/components/drops/DropsItem.tsx:119]: same here
```suggestion
                                        (isEditing || mappedColorSpace.value) ? '': 'tw-hidden',
```

File: packages/colors-block/src/components/drops/DropsItem.tsx [hunk 4]
```diff
@@ -87,50 +93,69 @@ export const DropsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
...
+                                        colorSpaceId={colorSpaceId}
+                                        onUpdate={onUpdate}
+                                    />
+                                ) : (
+                                    <Tooltip
+                                        withArrow
+                                        position={TooltipPosition.Right}
+                                        content={
+                                            <TooltipContent
+                                                colorName={color.name || ''}
```
Comment (celine-s) [inline packages/colors-block/src/components/drops/DropsItem.tsx:138]:
```suggestion
                                                colorName={color.name ?? ''}
```

File: packages/colors-block/src/components/TooltipContent.spec.ct.tsx [hunk 5]
```diff
@@ -6,30 +6,31 @@ import { TooltipContent } from './TooltipContent';
...
 const STATUS_IDLE_MESSAGE = 'Click to copy.';
 const STATUS_SUCCESS_MESSAGE = 'Copied!';
 const STATUS_ERROR_MESSAGE = 'Error copying. Try again.';
 
 describe('TooltipContent component', () => {
     it('renders a tooltip content component with idle status', () => {
-        mount(<TooltipContent colorValue={COLOR_VALUE} status="idle" />);
+        mount(<TooltipContent colorName={COLOR_NAME} colorValue={COLOR_VALUE} status="idle" />);
 
         cy.get(TooltipContentSelector).contains('Color Name');
```
Comment (celine-s) [inline packages/colors-block/src/components/TooltipContent.spec.ct.tsx:19]: could you use `COLOR_NAME`?

File: packages/colors-block/src/components/cards/CardsItemAdd.tsx [hunk 6]
```diff
@@ -10,21 +10,19 @@ export const CardsItemAdd = ({ colorSpaces, onConfirm }: ItemAddProps) => {
...
                 >
                     <IconPlus size={IconSize.Size24} />
                 </div>
             </ColorPickerFlyout>
 
             <div className="tw-pt-4 tw-px-6 tw-pb-5">
-                <div className="tw-w-[100px] tw-mb-3 tw-pointer-events-none tw-text-m tw-text-black-50 tw-font-bold">
-                    Color name
-                </div>
+                <div className="tw-mb-3 tw-pointer-events-none tw-text-m tw-text-black-50 tw-font-bold">Color name</div>
```
Comment (celine-s) [inline packages/colors-block/src/components/cards/CardsItemAdd.tsx:25]:
```suggestion
                <div className="tw-mb-3 tw-pointer-events-none tw-text-m tw-text-black-50 tw-font-bold">{COLOR_NAME}</div>
```

File: packages/colors-block/src/components/drops/DropsItemAdd.tsx [hunk 7]
```diff
@@ -12,15 +12,13 @@ export const DropsItemAdd = ({ colorSpaces, onConfirm }: ItemAddProps) => {
...
+                    className="tw-flex tw-justify-center tw-items-center tw-w-[100px] tw-h-[100px] tw-mx-auto tw-cursor-pointer tw-rounded-full tw-mb-3 tw-text-black tw-bg-button-background tw-shadow-inner-line tw-transition-all hover:tw-bg-button-background-hover hover:tw-shadow-inner-line-x-strong"
                 >
                     <IconPlus size={IconSize.Size24} />
                 </div>
             </ColorPickerFlyout>
 
-            <div className="tw-flex tw-w-[100px] tw-mb-3 tw-pointer-events-none tw-text-m tw-font-bold tw-text-black tw-text-center">
-                Color name
-            </div>
+            <div className="tw-mb-3 tw-pointer-events-none tw-text-m tw-font-bold tw-text-black-50">Color name</div>
```
Comment (celine-s) [inline packages/colors-block/src/components/drops/DropsItemAdd.tsx:21]:
```suggestion
            <div className="tw-mb-3 tw-pointer-events-none tw-text-m tw-font-bold tw-text-black-50">{COLOR_NAME}</div>
```

File: packages/colors-block/src/components/list/ListItem.tsx [hunk 8]
```diff
@@ -52,13 +53,18 @@ export const ListItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDe
                 <Tooltip
                     withArrow
                     position={TooltipPosition.Right}
-                    hoverDelay={0}
-                    content={<TooltipContent colorValue={mappedFirstColorSpace.value ?? ''} status={status} />}
+                    content={
+                        <TooltipContent
+                            colorName={color.name || ''}
```
Comment (celine-s) [inline packages/colors-block/src/components/list/ListItem.tsx:58]:
```suggestion
                            colorName={color.name ?? ''}
```

File: packages/colors-block/src/components/list/ListItem.tsx [hunk 9]
```diff
@@ -96,8 +103,19 @@ export const ListItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDe
                 {colorSpaces?.map((colorSpaceId) => {
                     const mappedColorSpace = mapColorSpaces(colorSpaceId, color);
 
+                    if (!isEditing && !mappedColorSpace.value) {
+                        return <></>;
```
Comment (celine-s) [inline packages/colors-block/src/components/list/ListItem.tsx:107]: do you need to return <></> or could you just `return;`?

File: packages/colors-block/src/components/list/ListItem.tsx [hunk 10]
```diff
@@ -115,14 +133,17 @@ export const ListItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDe
                                 <Tooltip
                                     withArrow
                                     position={TooltipPosition.Right}
-                                    hoverDelay={0}
                                     content={
-                                        <TooltipContent colorValue={mappedColorSpace.value ?? ''} status={status} />
+                                        <TooltipContent
+                                            colorName={color.name || ''}
```
Comment (celine-s) [inline packages/colors-block/src/components/list/ListItem.tsx:138]:
```suggestion
                                            colorName={color.name ?? ''}
```

**Review by celine-s — APPROVED** [review]
some small comments. Overall changes are quite straightforward. 👍

File: packages/colors-block/src/components/list/ListItem.tsx (hunk 9 above)
Comment (leko91) [reply packages/colors-block/src/components/list/ListItem.tsx:107]: Need to return something from a `.map`

File: packages/colors-block/src/components/ColorName.tsx [hunk 11]
```diff
@@ -30,6 +30,8 @@ export const ColorName = ({ viewType, initialColorName, isEditing, onBlur }: Col
                     value={colorName}
                     onChange={handleColorNameChange}
                     onBlur={onBlur}
+                    draggable={true}
+                    onDragStart={(event) => event.preventDefault()}
```
Comment (SamuelAlev) [inline packages/colors-block/src/components/ColorName.tsx:34]: Why?

Comment (leko91) [reply packages/colors-block/src/components/ColorName.tsx:34]: Simone asked that you can select the name and not drag when selecting

Comment (SamuelAlev) [reply packages/colors-block/src/components/ColorName.tsx:34]: then just `draggable={false}` no?

Comment (leko91) [reply packages/colors-block/src/components/ColorName.tsx:34]: That doesn't work 🤷‍♂️ Probably the dnd takes precedence, but not sure

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**